match/README.md

```diff
@@ -38,6 +38,8 @@ match
 
 A new approach to iOS code signing: Share one code signing identity across your development team to simplify your codesigning setup and prevent code signing issues.
 
+`match` is the implementation of the https://codesigning.guide concept. `match` creates all required certificates and profiles and puts them into a separate git repository. Every team member with access to the repo can use those certificates to code sign an app. Automatically repairs broken and expired profiles. This is all about having one repo per dev team.
```

Comment:
The last sentence sounds a bit strange, maybe something like:

"`match` creates all required certificates and provisioning profiles an stores them in a separate git repository. Every team member with access to the repo can use those credentials for code signing. `match` also automatically repairs broken and expired credentials. The easiest way to share signing credentials across teams"